Library/Formula/azure-cli.rb

```diff
@@ -0,0 +1,36 @@
+class AzureCli < Formula
+  desc "Official Azure CLI"
+  homepage "https://github.com/azure/azure-xplat-cli"
+  url "https://github.com/azure/azure-xplat-cli/releases/download/v0.9.13-December2015/azure-cli.0.9.13.tar.gz"
+  sha256 "f91e341c499624d6ba007ef3ad79d932c5b94a1f35acc49e3fbffd76c2317317"
+
+  head "https://github.com/azure/azure-xplat-cli.git", :branch => "dev"
+
+  option "without-completion", "Azure CLI bash completion will not be installed"
```

Comment:
It’s probably better not to provide this option and have a simpler `install` code. What are the use-cases of not installing the completion?